### PR TITLE
Add ability to disable environments

### PR DESCRIPTION
**Pull request by cpacey**

### CODE REVIEW

**Comment by cpacey** [comment]
I can include it in the other types if we want.  I wanted to get feedback on this first.

File: schemas/instance-boolean/v1_2.json [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+				"rules": {
+					"type": "array",
+					"title": "The sequential rules which are evaluated when not matched by an explicit target",
+					"additionalItems": false,
+					"minItems": 1,
+					"items": {
+						"$ref": "#/definitions/rule"
+					}
+				},
+				"disabled": {
```
Comment (cpacey) [inline schemas/instance-boolean/v1_2.json]: This is the only part added vs 1.1.

File: schemas/instance-boolean/v1_2.json [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+					"type": "boolean",
+					"title": "True if targeting should be enabled, false otherwise."
```
Comment (JeffAshton) [inline schemas/instance-boolean/v1_2.json]: English is backwards

Comment (JeffAshton) [reply schemas/instance-boolean/v1_2.json]: If true, the defaultVariation is always used, and the targets and rules are ignored.

File: schemas/instance-boolean/v1_2.json (hunk 1 above)
Comment (JeffAshton) [reply schemas/instance-boolean/v1_2.json]: I know.  I did a diff.  :)

**Pull request merged**